config/routes.rb

```diff
@@ -1,5 +1,10 @@
 Rails.application.routes.draw do
   devise_for :users
+
   root "dashboard#index"
   get "/", to: "dashboard#index"
+
+  resources :documents
```

Comment:
`documents`, `attendances` and `groups` are linked to a `user` aren't they?
If so, they probably should be a nested resource. Or is there any reason why they're not?

Reply:
Only attendances and groups but yes, they are nested resources.

Reply:
No idea what you mean by `responders for tournaments` 🤔 
TO be clear, is `document` a resources created bu users?

Reply:
I mean, A letter or file that the responsible person of the child get the legal responsibility if some accident happens during a tournament or something like that.
Document is a resource created only for user with role "teacher".

Reply:
Maybe, but I'm not sure how it will be match with the teacher resource and students resource (since they both are basically users) so, I prefer let outside in this feature, as I say, this is only a basic creation of the resources without any business logic.